src/NeptuneSchema.js

```diff
@@ -109,6 +122,31 @@ async function queryNeptuneSDK(query, params) {
     }
 }
 
+/**
+ * Queries neptune analytics graph using SDK (not to be used for neptune db).
+ */
+async function queryNeptuneGraphSDK(query, params = '{}') {
+    try {
+        const client = new NeptuneGraphClient({
```

Comment:
There is some non-trivial overhead to creating a new client for each query. I don't think this needs to be a blocker at the moment, however it should be noted that refactoring this to reuse a single client for the whole life of the pipeline should improve efficiency.

Reply:
Changed both sdk clients to be created only once per pipeline.